### DISCUSSION TITLE
deepin-draw: export dialog

### DISCUSSION THREAD

**Discussion by lidanger** [linuxdeepin/developer-center#2934] [Features Request & Ideas | 特性请求 & 头脑风暴]
When deepin-draw exports pictures, you can only input a single character in the filename box, and then the cursor will jump away. It can only be manually transferred back and then continue inputing.

In addition, can you set not to create a project, directly save as a picture, do not need export?

**Comment by BLumia** [comment]
> not to create a project, directly save as a picture, do not need export?

I guess we'll lose the ability to re-edit the drawboard after you save and quit the program :)
